assets/js/utilities/net.js

```diff
@@ -1,4 +1,4 @@
-import 'whatwg-fetch' // Polyfill for fetch
+import 'unfetch/polyfill' // Polyfill for fetch
```

Comment:
@developit switched us from `whatwg-fetch` to `unfetch/polyfill` to save some bytes.